src/useAnswersState.tsx

```diff
@@ -10,6 +10,10 @@ export type StateSelector<T> = (s: State) => T;
  */
 export function useAnswersState<T>(stateSelector: StateSelector<T>): T {
   const answers = useContext(AnswersHeadlessContext);
+  if (answers.state === undefined) {
+    throw new Error('Attempted to use AnswersHeadless before it\'s initialized.'
```

Comment:
maybe just `Attempted to call useAnswersState() outside an AnswersHeadlessProvider`? I think the first sentence is a little confusing because users might not know that AnswersHeadlessProvider is initializing AnswersHeadless